### DISCUSSION TITLE
Original data source and update frequency

### DISCUSSION THREAD

**Discussion by ryanbuckner** [TedJuch/TedTournament#12] [General]
This is truly amazing.  I was planning to build something similar but maybe I'll just use this.  It looks like each individual sheet looks at your master data sheet and pulls the data into the users' copy? 

What is the original source of your sheet ?   I have a [sheet](https://docs.google.com/spreadsheets/d/1WMczLzQzibHglB_kksK4b5JksXDDitUAyw5Y1HXMjog/edit#gid=139613902) that hits the ESPN API and pulls back data, but I haven't been able to determine how to filter it to only NCAA Tournament data. 

Are you using something simpler?

**Comment by TedJuch** [comment]
My master sheet is a combination of manual entry and some automatic data from [data.ncaa.com](https://data.ncaa.com/). For example:  https://data.ncaa.com/casablanca/scoreboard/basketball-men/d1/2023/03/09/scoreboard.json 
Hope that helps!
